pyproject.toml

```diff
@@ -13,14 +13,15 @@ requires-python = ">= 3.10"
 dependencies = [
     "ctakesclient >= 3, < 4",
     "delta-spark >= 2.3, < 3",
+    "fhirclient < 5",
```

Comment:
I know we just got rid of `fhirclient`, but it was the quickest/easiest way I could find to get python-friendly flattened versions of the FHIR spec.